dmsite/venv/src/Authentication/LoginTextField.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Created by christinadaley on 1/18/19.
+ */
+
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import {
+  Checkbox, Form, Icon, Input,
+} from 'antd';
+
+class TextField extends React.Component {
+    handleSubmit = (e) => {
+      e.preventDefault();
+      this.props.form.validateFields((err, values) => {
+        if (!err) {
+          console.log('Received values of form: ', values);
+        }
+      });
+    }
+
+    render() {
+      const { getFieldDecorator } = this.props.form;
+      return (
+        <Form onSubmit={this.handleSubmit} className="login-form">
+          <Form.Item>
+            {getFieldDecorator('email', {
+              rules: [{ required: true, message: 'Please input your email!' }],
+            })(
+              <Input prefix={<Icon type="user" style={{ color: 'rgba(0,0,0,.25)' }} />} placeholder="Email" />,
+            )}
+          </Form.Item>
+          <Form.Item>
+            {getFieldDecorator('password', {
+              rules: [{ required: true, message: 'Please input your Password!' }],
+            })(
+              <Input prefix={<Icon type="lock" style={{ color: 'rgba(0,0,0,.25)' }} />} type="password" placeholder="Password" />,
+            )}
+          </Form.Item>
+          <Form.Item>
+            {getFieldDecorator('remember', {
+              valuePropName: 'checked',
+              initialValue: true,
+            })(
+              <Checkbox> Remember me</Checkbox>,
+            )}
+          </Form.Item>
+          <Form.Item>
+            <a className="login-form-forgot" href="/createaccount">Forgot password </a>
+
+            {/* TO DO */}
+
+                    or
+            {' '}
+            <a href="/createaccount"> Create an account now!</a>
+          </Form.Item>
+          {/* <Form.Item>
+                    <Button type="primary" htmlType="submit" className="login-form-button">
```

Comment:
Remove this commented out code if it is not being used

Reply:
Can do! Thanks!